renpy/common/_shaders.rpym

```diff
@@ -157,6 +157,25 @@ init python:
         gl_FragColor = vec4(src.r * mask.r, src.g * mask.r, src.b * mask.r, mask.r);
     """)
 
+    renpy.register_shader("renpy.alphamask", variables="""
```

Comment:
Maybe add a comment before this and the previous one, saying which is for display.layout.AlphaMask and which is for movies, so that the code is more understandable for posterity.